### ISSUE TITLE
.m3u method for multi-disk isn't working

### ISSUE THREAD

**Issue by kirboi** [OpenEmu/OpenEmu#2992]
I've tried multiple times using the .m3u method for games with multiple discs, but it just doesn't work. I've tried using TextEdit AND TextMate but the result is the same -- OpenEmu doesn't recognize the file. Any idea of what I'm doing wrong? Btw the game I'm trying to play is The Legend of Dragoon... and it has 4 discs so to finish 1 but not be able to play the other 3 is quite agitating.

**Comment by clobber** [comment]
The procedure changed slightly for 2.0.4, so follow the [import guide](https://github.com/OpenEmu/OpenEmu/wiki/User-guide:-CD-based-games#q-i-have-a-multi-disc-game) very closely.

**Comment by Bleriott** [comment]
I'm having the same trouble. I've tried deleting the game files from the library to start from scratch. I've added the .m3u file with the below code:
Final Fantasy VIII (USA) (Disc 1).cue
Final Fantasy VIII (USA) (Disc 2).cue
Final Fantasy VIII (USA) (Disc 3).cue
Final Fantasy VIII (USA) (Disc 4).cue

I noticed in the guide (that has been linked in various threads) though about a file which has this code:
Folder containing all Final Fantasy VIII files/
└╴Final Fantasy VIII (Disc 1).bin
└╴Final Fantasy VIII (Disc 1).cue
└╴Final Fantasy VIII (Disc 2).bin
└╴Final Fantasy VIII (Disc 2).cue
└╴Final Fantasy VIII (Disc 3).bin
└╴Final Fantasy VIII (Disc 3).cue
└╴Final Fantasy VIII (Disc 4).cue
└╴Final Fantasy VIII (Disc 4).bin
└╴Final Fantasy VIII.m3u    <-- import this file

I just find the guide a bit misleading, am I meant to have one .m3u file in a folder containing the folders of each game disc (with their respective .bin and .cue files) or two .m3u files? Ive tried both but neither seems to work.

Any help would be greatly appreciated.
